src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,269 @@
+            '1451548 - Return 404 for indexes and namespaces that are expired',
+          tag: 'JS',
+          assignedto: 'None',
+          lastupdate: '2018-05-04',
+        },
+        {
+          project: 'Taskcluster',
+          description:
+            '1441977 - Run tests for taskcluster-treeherder in Taskcluster',
+          tag: 'JS',
+          assignedto: 'None',
+          lastupdate: '2018-05-03',
+        },
+        {
+          project: 'Servo',
+          description:
+            '1446768 - Only post "No taskcluster jobs.." to a PR once',
+          tag: 'JS',
+          assignedto: 'None',
+          lastupdate: '2018-05-01',
+        },
+        {
+          project: 'Taskcluster',
+          description:
+            '1457126 - Authorization failures should state which clientId lacks scopes',
+          tag: 'Rust',
+          assignedto: 'None',
+          lastupdate: '2018-04-05',
+        },
+        {
+          project: 'Taskcluster',
+          description: '44 - Add pagination to auth.listRoles',
+          tag: 'JS',
+          assignedto: 'None',
+          lastupdate: '2018-03-05',
+        },
+        {
+          project: 'Taskcluster',
+          description:
+            '1306494 - Add a diff+commit submit button to some text areas in tc-tools',
+          tag: 'JS',
+          assignedto: 'None',
+          lastupdate: '2018-02-12',
+        },
+        {
+          project: 'Taskcluster',
+          description:
+            '333 - Make azure-entities and azure-blob0-storage independent of tc-lib-monitor',
+          tag: 'JS',
+          assignedto: 'None',
+          lastupdate: '2018-02-04',
+        },
+        {
+          project: 'Taskcluster',
+          description: '1344912 - Support tag events, too',
+          tag: 'JS',
+          assignedto: 'None',
+          lastupdate: '2018-01-01',
+        },
+        {
+          project: 'Taskcluster',
+          description:
+            '1453714 - Return http 424 instead of 403 for error artifacts',
+          tag: 'Python',
+          assignedto: 'None',
+          lastupdate: '2017-12-04',
+        },
+      ].sort((a, b) => (a.lastupdate > b.lastupdate ? -1 : 1)),
+      page: 0,
+      rowsPerPage: PROJECTS_PAGE_SIZE,
+    };
+  }
+
+  handleRequestSort = (event, property) => {
+    const orderBy = property;
+    let order = 'desc';
+
+    if (this.state.orderBy === property && this.state.order === 'desc') {
+      order = 'asc';
+    }
+
+    const data =
+      order === 'desc'
+        ? this.state.data.sort((a, b) => (b[orderBy] < a[orderBy] ? -1 : 1))
+        : this.state.data.sort((a, b) => (a[orderBy] < b[orderBy] ? -1 : 1));
+
+    this.setState({ data, order, orderBy });
+  };
+
+  handlePageChange = (event, page) => {
+    this.setState({ page });
+  };
+
+  render() {
+    const { classes } = this.props;
+    const { data, order, orderBy, rowsPerPage, page } = this.state;
+
+    return (
+      <Paper className={classes.root}>
```

Comment:
Replace `<Paper className={classes.root}>` with `<div className={classes.root}>`. I don't think it's needed.